resource-types/typescript-example-website-monitor/package.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "example-monitoring-website",
+    "version": "0.1.0",
+    "description": "AWS custom resource provider named Example::Monitoring::Website.",
+    "private": true,
+    "main": "dist/handlers.js",
+    "files": [
+        "dist"
+    ],
+    "scripts": {
+        "build": "npx tsc",
+        "prepack": "npm run build",
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "engines": {
+        "node": ">=12.0.0",
+        "npm": ">=6.9.0"
+    },
+    "dependencies": {
+        "cfn-rpdk": "npm:@org-formation/cfn-rpdk@^0.5.0",
```

Comment:
@craigataws We should be using the GA version: https://www.npmjs.com/package/@amazon-web-services-cloudformation/cloudformation-cli-typescript-lib